docs/conf.py

```diff
@@ -68,6 +76,8 @@
     "package_title": project,
     "package_version": release,
     "package_repo": 'https://github.com/compas-dev/compas_fab',
+    "package_docs": package_docs_root,
+    "package_old_versions": package_docs_versions
```

Comment:
Doesn't "package_old_versions" also contain "latest"?  Is that already old?

Reply:
Maybe a better name would have been `package_all_versions` 😛  (this variable name is [defined in the theme](https://github.com/compas-dev/sphinx_compas_theme/blob/master/sphinx_compas_theme/compaspkg/theme.conf))